mathics/builtin/specialfns/bessel.py

```diff
@@ -79,7 +79,7 @@ class AiryAiPrime(_MPMathFunction):
         "Derivative[1][AiryAiPrime]": "(#1 AiryAi[#1])&",
     }
 
-    summary_text = "Derivative of the Airy function Ai"
+    summary_text = "Derivative of the Airy's function Ai"
```

Comment:
Capital letter here and in other places.

Reply:
fixed